cpp_tests/include/clink/cpp_tests/test_util.h

```diff
@@ -90,10 +90,9 @@ void saveMetaDataModelData(std::string dir_name, nlohmann::json params,
 
 // Creates a ClinkRunner, executes the provided mlir script and returns the
 // execution result.
-llvm::SmallVector<RCReference<AsyncValue>, 4>
-runMlirScript(tfrt::HostContext *host_context, MLIRContext *mlir_context,
-              string_view mlir_script,
-              llvm::ArrayRef<RCReference<AsyncValue>> inputs) {
+llvm::SmallVector<RCReference<AsyncValue>, 4> runMlirScript(
+    tfrt::HostContext *host_context, MLIRContext *mlir_context,
```

Comment:
In previous discussions in #8 ([comment link](https://github.com/flink-extended/clink/pull/8#discussion_r787332533)), we have wanted to change codes like `HostContext *` to `HostContext* ` in order to comply with TFRT's style. But as the discussion have pointed out, simply changing the clang-format style from LLVM to Google cannot make it, due to the Google style configuration. In order to solve such problems we may need to make a one-time manual change to all pointer usages in C++ codes. I think it worth applying the manual work, but would like my idea to be confirmed before making the change.

Reply:
Sounds great!